README.md

```diff
@@ -1,56 +1,38 @@
+Or compile it yourself: `go get github.com/legolord208/DiscordConsole`, you'll also need the DiscordGo development version; to get that type `make dgo`. Then, actually make the switch to develop: `git checkout develop`. Finally, just type `go install`.
 
-**If** you want to get the absolutely latest development update, you'll have to compile it yourself.  
-Do that using
-```
-go get github.com/legolord208/DiscordConsole
-```
-.
+### Usage
 
-You'll also need the DiscordGo development version. To get that, type `make dgo`.  
-Then, actually make the switch to develop:
-```
-git checkout develop
-```
-Then just type `make`, or `go install`. You choose.
+It is recommended to run DiscordConsole in a Terminal. To do that, just `cd` to the folder you extracted it and run it by typing `DiscordConsole`. There are a few launch options covered in the wiki. *Soon™*
 
-## Special features
-Set playing status, simulate typing, bulk delete and more.
+When starting the DiscordConsole from Windows, you might get a screen saying that Windows SmartScreen prevented an unrecognized app from starting. Just click the **More Info** then **Run anyway**.
 
-In addition, you are even able to delete bot defined roles.  
-If you try to do this in discord, it just says:  
-![Imgur](http://i.imgur.com/Ubr2OMZ.png)
+Once started, you'll get asked to fill in your token. If you don't know how to get it, check this wiki page. *Soon™*
+For the first setup, we will use an user account. Setups with bot accounts and webhook are explained on the wiki as well. *Soon™*
 
-This was also discovered recently, by a friend of mine:  
-You can bypass the black background in an avatar when setting the bot avatar.  
-![Imgur](http://i.imgur.com/Q0GQR8d.png)
+Once you got your token, just type `user TOKEN` in the terminal. You'll get the following screen:
+![Picture](https://i.imgur.com/KPCVmlH.png)
 
-### Bulk delete
-DiscordConsole lets you bulk delete messages. This allows you to delete a bunch of messages at once, without needing to write any code!
+Try to type something like `help` and let the magic happens! For a detail review of each commands, check this wiki page *Soon™* or type `help command`.
 
-### Log
-Log the last 100 messages to a file, or just view the last 10 directly in the console!
+### Contribute
 
-## Command line
-DiscordConsole has full command line support. You can supply a bot/user token or email/password on the command line, and even specify commands to run, so you can use DiscordConsole for scripting.
+Feels free to submit a Pull Request! Not a coder, but want to be involved in the project? Send us some suggestions trough isssues.
 
-Having it start with a specific server automatically selected? Making a cron job to automatically message how many days until Trump leaves? Easy!  
-![Imgur](http://i.imgur.com/2mst4pH.png)  
+### License
+
+GNU G.P.L. © Legolord208 - [Click here for more informations.](https://github.com/legolord208/DiscordConsole/blob/master/LICENSE)
```

Comment:
Small nitpick, but could you keep the casing consistent and put `Lego` in capitals? Thanks!